client/src/views/Home/CreateParty.jsx

```diff
@@ -59,6 +59,7 @@ function CreateGame({
           label="Teams"
           subLabel="The number of teams playing"
           value={teamsCount}
+          isNumberVariant={true}
```

Comment:
with react. If a prop is just supposed to be true you can omit the `={true}` part. 

```
<TextInput
  isNumberVariant
  name="teams-count"
/>
```

Note: for cases like this I prefer to put these props first. Doesn't really do anything but it's cleaner to me.

Reply:
that makes sense, and actually clears up some confusion i had about the flow of information in this game. a little confusing since it looks like a reference to a variable, but not if you're used to it i bet.